client/source/components/questionsAnswers/AnswerItem.jsx

```diff
@@ -2,18 +2,23 @@ import React from 'react';
 import styled from 'styled-components';
 import { format, parseISO } from 'date-fns';
 import HelpfulA from './HelpfulA';
+import AnswerPhoto from './AnswerPhoto';
 
 const AnswerItem = function CreateAnswerItemComponent({
-  id, answer, username, date, getQuestions, helpfulness
+  id, answer, username, date, getQuestions, helpfulness, photos
 }) {
   const parsedDate = parseISO(date);
   const formattedDate = format(parsedDate, 'PPP');
   return (
     <>
       <Answer>
-        <h4>A:</h4>
-        <p>{answer}</p>
+        <H4>A:</H4>
+        <P>{answer}</P>
       </Answer>
+      <APhotos>
+        {/* eslint-disable-next-line react/no-array-index-key */}
+        {photos.length ? (photos.map((photo, i) => (<AnswerPhoto key={i} photo={photo} />))) : null}
```

Comment:
You **really** want to include keys for anything that React sees via a map. This can super bite you in the bottom instrange ways. Feel free to use `photo.src` or something long in here, since the machine is right that a simple index is not enough.

Reply:
Sounds good. There was no other property in photos, so I used a slice of the url that has an identifying number in it. I received no complaints from the console.